radiomics/__init__.py

```diff
@@ -161,6 +161,18 @@ def getInputImageTypes():
   return _inputImages
 
 
+def getProgressFunctions():
```

Comment:
should that function be private: `_getProgressFunctions()` ? Or should the variable `(init|report|close)progress` be private global ?

Either way we need to document the signature of the functions too.